dts/bindings/clock/nordic,nrf53x-lfxo.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: Nordic nRF low-frequency crystal oscillator (nRF53X series)
+
+compatible: "nordic,nrf53x-lfxo"
```

Comment:
It's nRF53 Series, not nRF53X, so I think the compatibles should be "nordic,nrf53-*". Actually, we already have "nordic,nrf53-flash-controller" in tree.

Reply:
we actually have `SOC_SERIES_NRF53X`, also to be consistent with 52 where we have 52x.

Reply:
So please let me know what you prefer, I have no strong opinion here.

Reply:
For dts I'm not bothered either way, the problem is with the Kconfig symbol. Can see here the series is nrf53 https://github.com/zephyrproject-rtos/zephyr/blob/main/soc/nordic/soc.yml#L16 which means https://github.com/zephyrproject-rtos/zephyr/blob/main/soc/nordic/nrf53/Kconfig.soc#L6 should be SOC_SERIES_NRF53 not SOC_SERIES_NRF53X, this will completely break when the symbols are automatically generated by the build system, it hasn't been changed yet because that symbol is used in a lot of places, including the HAL, so that will need a big update. I would lean on nrf53 for dts, since nrf53x is an issue with the Kconfig symbol

Reply:
updated to `nrf53`